src/foam/u2/view/ScrollTableView.js

```diff
@@ -85,6 +85,15 @@
       documentation: `
         A reference to the table element we use in the fitInScreen calculations.
       `
+    },
+    {
+      type: 'Int',
+      name: 'accumulator',
+      value: 0,
```

Comment:
0 is already the default.

Reply:
I thought so, but if I remove this line it breaks scrolling.

However, I just tried changing line 90 from `type: 'Int'` to `class: 'Int'` and that worked without the value being set explicitly.